static/app/views/issueDetails/header.tsx

```diff
@@ -209,6 +211,14 @@ function GroupHeader({
     };
   }, [location, baseUrl]);
 
+  const replayRoute = useMemo(() => {
+    const searchTermWithoutSort = omit(location.query, ['sort']);
```

Comment:
Looking at the diff, should it be location.search? What's the difference?

Reply:
`location.search` is a string but omit takes in an object

Reply:
(`search` is kinda like the stringified version of `query`)